src/openclsim/critical_path/simulation_graph.py

```diff
@@ -37,12 +41,53 @@ class SimulationGraph:
         should appear in the list as tuple ``(1, 2)``.
     """
 
+    # name of added columns to the recorded_activities_df
+    __COLNAME_CRITICAL = "is_critical"
+
+    # mapping from column name to attribute
+    __SUPERLOG_COLUMNS = {
```

Comment:
Superlog is not the name we use anymore. Seems right to change

Reply:
check